.github/container/Dockerfile.base

```diff
@@ -37,7 +36,14 @@ apt-get install -y \
   python3-pip \
   rsync \
   vim \
-  wget
+  wget \
+  lsb-release software-properties-common  # for llvm.sh
+
+# Install LLVM/Clang 17
```

Comment:
Pedantic nit: the comment says 17 but the code is generic.

Reply:
Will fix before merging (if I don't forget :grin:)